apps/server/src/app/export.controller.ts

```diff
@@ -182,6 +184,40 @@ export class ExportController extends BaseController {
     }
   }
 
+  /**
+   * The publisherVersion() returns an array of publisher versions from the sonatype.org endpoint defined below.
+   */
+  @Get(':publisher-version')
+  public publisherVersion(): Promise<any> {
+    const url = 'https://oss.sonatype.org/service/local/repositories/snapshots/index_content/?groupIdHint=org.hl7.fhir.publisher&artifactIdHint=org.hl7.fhir.publisher&';
+
+    return this.httpService.get(url, this.getOptions()).toPromise()
+      .then((results) => {
+        const versions = [];
+        if (results.data) {
+          let arr = [];
+          arr = results.data['data']['children'][0]['children'][0]['children'][0]['children'][0]['children'][0]['children'].sort().reverse();
+          arr.forEach(function(object) {
+            versions.push(object['nodeName']);
+          })
+        }
+        const sortedVersions = versions.sort().reverse();
+        const currentVersion = sortedVersions[0];
+        sortedVersions[0] = currentVersion + ' (Current)';
+        return sortedVersions;
+      }).catch(function(reason) {
+        return reason;
+      })
+  }
+
+  private getOptions() {
```

Comment:
This is not used anywhere else as far as I can tell... would rather just create the options directly in the getPublisherVersions() method.

Reply:
agreed